tests/gordo/machine/test_machine.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+import peewee
+
+from gordo.machine import Machine
+from gordo.reporters.postgres import PostgresReporter, Machine as PostgresMachine
```

Comment:
Maybe good to have `postgres.Machine` just be `postgres.PostgresMachine` on the regular?

Reply:
I think the point here is that `Machine` to a `Reporter` is independent of our interpretation. It's a database model representing a gordo machine. In that respect, it seems silly to mandate that users would prefix their database models with name of their database providers. I admit it's slightly awkward to import for a _test_ but it's not used anywhere inside the core of gordo.